### DISCUSSION TITLE
【已解决】【MAC】maa突然不能用了，尝试按照教程操作，也不行。

### DISCUSSION THREAD

**Discussion by RavenFL** [MaaAssistantArknights/MaaAssistantArknights#7120] [Q&A] [answered]
事情已经解决，不知道能否给到大家参考意见。
导致mac版本的maa不能用的原因是，在刷理智的设置里面，不可以自主输入关卡，但凡自主输入关卡，maa就会出现不运作的情况。
<img width="936" alt="image" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/149344566/07b77c41-d862-4a5c-a5c6-bf6cd0d81c61">
事实上，在出现问题之前的几个月里面，这个还是能正常运作的，因此不太清楚这是我自己的问题还是普遍问题。

------------------------------------------------
RT，用的mac os。
上周突然某天开始maa不能使用，playcover和maa的配置都没有修改过。
不能用的具体表现在于说，点击maa的任务开始后，呈现下图画面。同时，游戏界面没有动作，直接原地不动了。
<img width="936" alt="image" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/149344566/5292d7da-c02f-4674-a7e7-bc96dc9ad34c">
然后按照教程提示重新安装了最新版本的playcover也依旧不行，需要提供额外信息的话烦请大佬们回复，感谢各位大佬。

**Comment by ABA2396** [comment, marked as answer]
LS也没有-10啊）
[一键长草-刷理智](https://maa.plus/docs/1.1-详细介绍.html#一键长草-刷理智)
